pkgs/desktops/gnome-3/extensions/volume-mixer/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, glib, pulseaudio, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-volume-mixer";
+  version = "3.38.0";
+
+  src = fetchFromGitHub {
+    owner = "aleho";
+    repo = "gnome-shell-volume-mixer";
+    rev = version;
+    sha256 = "12iqw1ggzqjgnb6pnp32z8d847wx2sbiz5dl316cgmc2pzn9anj5";
+  };
+
+  patches = [
+    ./find_libpulse.patch
+  ];
+
+  buildInputs = [
+    glib
```

Comment:
If this is added for `glib-compile-schemas`, it should go to `nativeBuildInputs`, ideally with a comment.